pom.xml

```diff
@@ -127,14 +127,14 @@
         <dependency>
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
-            <version>9.4.1208</version>
+            <version>9.4.1209</version>
             <scope>test</scope>
         </dependency>
 
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
-            <version>5.1.39</version>
+            <version>6.0.3</version>
```

Comment:
According to https://dev.mysql.com/downloads/connector/j/, version 6.0.3 is still a development release. The latest GA release is 5.1.39.